contracts/tests/conftest.py

```diff
@@ -183,36 +183,74 @@ def block_header_by_malicious_non_validator(malicious_non_validator_key):
 
 
 @pytest.fixture(scope="session")
-def real_price_validator_auction_contract(deploy_contract, whitelist):
-    contract = deploy_contract("ValidatorAuction")
+def validator_auction_contract(deploy_contract, whitelist, web3, default_account):
+    deposit_locker = deploy_contract("DepositLocker")
+    contract = deploy_contract(
+        "TestValidatorAuctionFixedPrice", constructor_args=(deposit_locker.address,)
+    )
+    deposit_locker.functions.init(
+        web3.eth.blockNumber + 50,
+        "0x0000000000000000000000000000000000000000",
+        contract.address,
+    ).transact({"from": default_account})
+
     add_whitelist_to_validator_auction_contract(contract, whitelist)
 
     return contract
 
 
 @pytest.fixture(scope="session")
-def no_whitelist_validator_auction_contract(deploy_contract):
-    contract = deploy_contract("TestValidatorAuctionFixedPrice")
+def real_price_validator_auction_contract(
+    deploy_contract, whitelist, default_account, web3
+):
+    deposit_locker = deploy_contract("DepositLocker")
+
+    contract = deploy_contract(
+        "ValidatorAuction", constructor_args=(deposit_locker.address,)
+    )
+    deposit_locker.functions.init(
+        web3.eth.blockNumber + 50,
+        "0x0000000000000000000000000000000000000000",
```

Comment:
Please give a name to the arguments so we don't have to go to the contract to see what they are.

Reply:
534398b935d5fc87d8f2bbccddf227d7a5a7c5a3